test/extract_method_baseline/test_example_from_paper.py

```diff
@@ -27,6 +25,21 @@ def test_semantic_extraction(self):
                 f"Comparison index is {comparison_index}.",
             )
 
+    def test_statements_clustering(self):
+        self.maxDiff = None
+        semantic = extract_method_statements_semantic(self.method_ast)
+        clusters = cluster_statements(semantic)
+
+        for cluster_index, (actual_cluster, expected_cluster) in enumerate(
+            zip_longest(clusters, self.expected_clusters)
+        ):
+            actual_cluster_statement_types = [statement.node_type for statement in actual_cluster]
+            self.assertEqual(
+                actual_cluster_statement_types,
+                expected_cluster,
+                f"Failed on {cluster_index}th cluster comparison",
+            )
```

Comment:
Is it correct indentation here?

Reply:
Do you mean correct as able to run this file, or from some other demands? I use black autoformatter and this fie was formatted by it.